src/kprop/kpropd.c

```diff
@@ -1401,18 +1401,16 @@ recv_database(krb5_context context, int fd, int database_fd,
         if (krb5_is_krb_error(&inbuf))
             recv_error(context, &inbuf);
         retval = krb5_rd_priv(context, auth_context, &inbuf, &outbuf, NULL);
+        krb5_free_data_contents(context, &inbuf);
```

Comment:
This seems out of scope.  Accidentally pulled in?

Reply:
Yeah, technically we just need to move outbuf downward, i'll remove.